tests/unit/lang/translator/MOfilesTesting/en.txt

```diff
@@ -152,7 +152,7 @@ Print fan not spinning. Check it for possible debris, then inspect the wiring.
 Print Time
 Print will end
 PRINTING
-Printing time
+Elapsed
```

Comment:
Please remove changes in the MOfilesTesting, these files are used for testing the translation provider and they dont reflect current state of translation in firmware